enterprise/app/code/code_v2.tsx

```diff
@@ -266,15 +266,15 @@ export default class CodeComponentV2 extends React.Component<Props, State> {
 
   ticketsForPosition(pos: monaco.Position): string[] {
     const refs = this.getKytheRefsForRange(new monaco.Range(pos.lineNumber, pos.column, pos.lineNumber, pos.column));
-    if (!refs) {
+    if (!refs?.length) {
```

Comment:
seems like we're not actually relying on the return value of `undefined` for anything meaningful? you could just make getKytheRefsForRange always return an array and make this a smidge less clunky